### PR TITLE
Remove try/catch & change var to let

### PR DESCRIPTION
**Pull request by virkt25**
Remove try/catch block as per feedback as it is unnecessary & changed
var to let as per coding standards.

### CODE REVIEW

File: services/account/controllers/AccountController.ts [hunk 1]
```diff
@@ -12,27 +12,15 @@ export class AccountController {
   }
 
   async getAccount(filter) {
-    try {
```
Comment (ssh24) [inline services/account/controllers/AccountController.ts]: Why take off the try/catch block? If we are taking it off the try/catch block, then we are either assuming the **filter** parameter is an object or there are further checks in the `this.repository.find` function to validate the parsed filter is an object.

Comment (superkhau) [inline services/account/controllers/AccountController.ts]: See https://github.com/strongloop/loopback-next-example/pull/61#discussion_r123864484

Comment (ssh24) [inline services/account/controllers/AccountController.ts]: Ah, I see. Discard my comments then. :)

Comment (virkt25) [inline services/account/controllers/AccountController.ts]: Currently the filter object is coming in as a string. It's converted to an object still but as Raymond pointed out [here](https://github.com/strongloop/loopback-next-example/pull/61#pullrequestreview-46096320) the promise will be rejected if there is an error and hence we don't need an explicit rejection using try / catch.

**Pull request merged**